geometry_msgs/Point offset

